bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java

```diff
@@ -399,14 +399,15 @@ private void configureCredentialProvider(StubSettings.Builder stubSettings) thro
 
   private Credentials buildCredentialFromPrivateKey(
       String serviceAccountEmail, String privateKeyFile) throws IOException {
+
     try {
+      KeyStore keyStore = KeyStore.getInstance("PKCS12");
+
+      try (FileInputStream fin = new FileInputStream(privateKeyFile)) {
+        keyStore.load(fin, "notasecret".toCharArray());
+      }
       PrivateKey privateKey =
-          SecurityUtils.loadPrivateKeyFromKeyStore(
```

Comment:
I think we should continue to use this dependency here. I don't want to maintain this code in two places.

Reply:
As discussed offline, I think taking on an external dependency for 3 lines of code is too high of a cost to maintain.